.github/workflows/ci.yaml

```diff
@@ -0,0 +1,125 @@
+name: CI
+
+on: [push]
+
+jobs:
+  cvc4:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Inspect
+      run: |
+         echo $PWD
+         find .
+    - name: Get CVC4
+      run: |
+        git clone -b eqrange-quant --single-branch https://github.com/GaloisInc/vadd-CVC4.git .
```

Comment:
Seems like this could be replaced with [actions/checkout@v2](https://github.com/actions/checkout#Checkout-a-different-branch) which would be the cannonical way to do it.

Reply:
Are you sure?  This is a different repo than the main one.

Reply:
It looks like I can.

Reply:
I'll consider changing this, but what does it buy me?

Reply:
Sorry for the late reply, I didn't get a notification somehow.

Using the actions is preferred whenever possible because keeping with the conventional way to do something reduces cognitive overhead should someone else want to look at the code and contribute. Actions also strive to be cross platform while arbitrary shell commands don't.

I'm not particularly worried about it. But decreasing technical complexity by using established conventions has a tangible benefit for pipelines, especially if you ever want to test on windows as well.

Reply:
When you say "Using the action is prefered", who is the subject?  i.e., Is this Galois IT speaking, yourself, or some other entity?

Reply:
I disagree that there is a consensus, but I've changed to accommodate.